example/src/main/java/com/attentive/example/ExampleApp.java

```diff
@@ -12,7 +12,8 @@ public class ExampleApp extends Application {
     // The mode in which to run the Attentive Android SDK
     private static final AttentiveConfig.Mode MODE = AttentiveConfig.Mode.PRODUCTION;
 
-    public AttentiveConfig attentiveConfig;
+    private AttentiveConfig attentiveConfig;
+    public AttentiveEventTracker attentiveEventTracker;
```

Comment:
AttentiveEventTracker is a global singleton, so no need to have an instance variable of it

Reply:
I originally added this variable, so it probably snuck back in somehow

Reply:
ah gotcha, will remove!